README.md

```diff
@@ -171,6 +171,16 @@ Sometimes it is handy to have public access to Redshift clusters (for example if
 
 It is possible to integrate this VPC module with [terraform-aws-transit-gateway module](https://github.com/terraform-aws-modules/terraform-aws-transit-gateway) which handles the creation of TGW resources and VPC attachments. See [complete example there](https://github.com/terraform-aws-modules/terraform-aws-transit-gateway/tree/master/examples/complete).
 
+## Network Firewall Configuration
+If you need to use the AWS Network Firewall to provide ingress or egress restrictions that can't be accommodated easily
+by security groups, you can enable it using the following arguments:
+```hcl
+enable_networkfirewall     = true
+networkfirewall_subnets    = ["10.0.11.0/24", "10.0.12.0/24", "10.0.13.0/24"]
```

Comment:
Consider changing these to /28s as this is what amazon documentation suggests.  /28s are also already used in the example files elsewhere in the diff.

Reply:
updated thanks